installing_on_kubernetes/_topics/external_messaging.md

```diff
@@ -0,0 +1,6 @@
+## Configuring external messaging
+
+It is possible to run with an external Kafka messaging server. To do this, create the required secret with the correct parameters using the folowing example and provide that secret name as `kafkaSecret` in the CR.
+```bash
+$ kubectl create secret generic kafka-secrets --from-literal=hostname=your.fqdn --from-literal=username=your-username --from-literal=password=your-password 
```

Comment:
For  now, since the whole doc is `oc`, should this also be `oc`?
```suggestion
$ oc create secret generic kafka-secrets --from-literal=hostname=your.fqdn --from-literal=username=your-username --from-literal=password=your-password 
```

Reply:
Separately, let's use `<>` like we do in other examples:

```suggestion
$ kubectl create secret generic kafka-secrets --from-literal=hostname=<your FQDN> --from-literal=username=<your username> --from-literal=password=<your password>
```